### PR TITLE
Bundle Gemfile and Gemfile.lock

### PR DESCRIPTION
**Pull request by kenhys**
Closes: #128

### CODE REVIEW

**Comment by kenhys** [comment]
```
% rpm -ql ./td-agent/yum/repositories/centos/8/x86_64/Packages/td-agent-4.0.1-1.el8.x86_64.rpm G td-agent/share/G
/opt/td-agent/share/Gemfile
/opt/td-agent/share/Gemfile.lock
```

```
% dpkg-deb -c ./td-agent/apt/repositories/debian/pool/buster/main/t/td-agent/td-agent_4.0.1-1_amd64.deb G share/G
-rw-r--r-- root/root      2049 2020-08-07 18:03 ./opt/td-agent/share/Gemfile
-rw-r--r-- root/root      5742 2020-08-07 18:03 ./opt/td-agent/share/Gemfile.lock
```

**Comment by kenhys** [comment]
I'll check windows build.

File: td-agent/Rakefile [hunk 1]
```diff
@@ -723,6 +728,21 @@ class BuildTask
     cp(src, dest)
   end
 
+  def install_gemfiles
+    ensure_directory(staging_sharedir) do
+      ["Gemfile", "Gemfile.lock"].each do |file|
+        if windows?
+          source_dir = File.join(__dir__, "..", "gemfiles", file == "Gemfile" ? "linux" : "windows")
```
Comment (cosmo0920) [inline td-agent/Rakefile:735]: Can we unity `windows?` if clause? Or, split for readability?

**Comment by kenhys** [comment]
image

File: td-agent/Rakefile (hunk 1 above)
Comment (kenhys) [reply td-agent/Rakefile]: Fixed.

**Review by cosmo0920 — APPROVED** [review]
LGTM!

**Pull request merged**